src/panels/lovelace/cards/hui-area-card.ts

```diff
@@ -54,6 +54,14 @@ export const DEVICE_CLASSES = {
   binary_sensor: ["motion", "moisture"],
 };
 
+const SUM_DEVICE_CLASSES = [
+  "power",
+  "apparent_power",
+  "reactive_power",
+  "current",
+  "gas",
```

Comment:
If we look at sensor classes that can be only TOTAL type, maybe also: 
- energy
- monetary
- reactive energy
- volume
- water